Added an FAQ for different downloading options (#27)
stjudecloud/university: docs/genomics-platform/faq/index.md

section](../requesting-data/how-to-fill-out-daa/#the-data-access-agreement)
for any data sets you want to download.

> ## Is there another mechanism for downloading the data besides using the dx command line tool?

Reviewer: ```suggestion
## Is there another mechanism for downloading the data besides using the `dx` command line tool?
```

> For an out of the box solution, the dx download command is the canonical way to download files from St. Jude Cloud Genomics Platform. If you are downloading large amounts of files, you may also try the DNAnexus Download Agent, which at times has been reported to be more performant and/or reliable.

Reviewer: Can you link the DX download agent?

> For an out of the box solution, the dx download command is the canonical way to download files from St. Jude Cloud Genomics Platform. If you are downloading large amounts of files, you may also try the DNAnexus Download Agent, which at times has been reported to be more performant and/or reliable.

Reviewer: ```suggestion
For an out of the box solution, the `dx` download command is the canonical way to download files from St. Jude Cloud Genomics Platform. If you are downloading large amounts of files, you may also try the [DNAnexus Download Agent](https://documentation.dnanexus.com/downloads#download-agent), which at times has been reported to be more performant and/or reliable.
```

> St. Jude Cloud data is made available via signed links that are generated by Azure Storage and expire after a certain amount of time. This mechanism is the standard and most broadly available method for accessing data stored in the cloud environments, and the files may be accessed by a number of open source tools for download. To generate a signed link to any of the files, you may make use of the dx make_download_url command followed by any of the major command line tools (aria2c, wget, curl, etc). Note that, if you choose to generate signed links to our data, it is your responsibility to ensure the links remain secure, including but not limited to the links remaining only known by yourself (so that others can not access the data in an authorized manner, as anyone can use them) and expiring as soon as possible (to minimize the amount of time the link is live).

Reviewer: ```suggestion
St. Jude Cloud data is made available via signed links that are generated by Azure Storage and expire after a certain amount of time. This mechanism is the standard and most broadly available method for accessing data stored in the cloud environments, and the files may be accessed by a number of open source tools for download. To generate a signed link to any of the files, you may make use of the `dx make_download_url` command followed by any of the major command line tools (`aria2c`, `wget`, `curl`, etc). Note that, if you choose to generate signed links to our data, it is your responsibility to ensure the links remain secure, including but not limited to the links remaining only known by yourself (so that others can not access the data in an authorized manner, as anyone can use them) and expiring as soon as possible (to minimize the amount of time the link is live).
```

No commercial solution (such as aspera) is available, nor are any custom transfer processes.

> We note here that downloading data at a large scale, no matter which tool is used, is an inherently unpredictable and unreliable process. This is a key driving factor for why so many in the community (including ourselves with the St. Jude Cloud project) have adopted the model of bringing tools to the data in the cloud. While we allow users to download data from our platform, we strongly recommend those having issues with downloading data analyze the data in the cloud. 

Reviewer: ```suggestion
We note here that downloading data at a large scale, no matter which tool is used, is an inherently unpredictable and unreliable process. This is a key driving factor for why so many in the community (including ourselves with the St. Jude Cloud project) have adopted the model of bringing tools to the data in the cloud. While we allow users to download data from our platform, we recommend those having issues with downloading data analyze the data in the cloud or try downloading the data again at a later time with retries enabled.
```
